Exercise_2.java

```diff
@@ -8,12 +8,33 @@
        of pivot */
     void swap(int arr[],int i,int j){
         //Your code here   
+
+       arr[i] = arr[i] * arr[j];
+       arr[j] = arr[i] / arr[j];
+       arr[i] = arr[i] / arr[j];

```

Comment:
good approach
but what if the numbers are very large?

Reply:
Have Added the change please review!